subiquitycore/tests/__init__.py

```diff
@@ -6,6 +6,12 @@
 
 
 class SubiTestCase(IsolatedAsyncioTestCase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Enable calling system_scripts from tests
+        if "SNAP" not in os.environ:
+            os.environ["SNAP"] = "."
```

Comment:
I feel like we should manage this closer to the test, the test case class could be fine.

Reply:
I changed this to have the test classes that needed it modify the environment using `setUpClass` and then put the original back in `tearDownClass`, which I liked better. Wdyt?

Reply:
https://stackoverflow.com/questions/31582750/python-mock-patch-os-environ-and-return-value maybe?

Reply:
Ah yeah that's better for sure. Done.